libsqlite3-sys/Cargo.toml

```diff
@@ -13,13 +13,13 @@ categories = ["external-ffi-bindings"]
 
 [features]
 default = ["min_sqlite_version_3_14_0"]
-bundled = ["cc", "bundled_bindings"]
-bundled-windows = ["cc", "bundled_bindings"]
+bundled = ["dep:cc", "bundled_bindings"]
+bundled-windows = ["dep:cc", "bundled_bindings"]
 bundled-sqlcipher = ["bundled"]
-bundled-sqlcipher-vendored-openssl = ["bundled-sqlcipher", "openssl-sys/vendored"]
-buildtime_bindgen = ["bindgen", "pkg-config", "vcpkg"]
+bundled-sqlcipher-vendored-openssl = ["bundled-sqlcipher", "dep:openssl-sys", "openssl-sys?/vendored"]
```

Comment:
the question mark is useless, no ?

Reply:
I think its a good practice to have it because it signifies that the `openssl-sys` crate is optional, and if someone removes the `dep:openssl-sys` later on, or moves it to be part of `bundled-sqlcipher` or some other feature, and later gets lost somehow, it does not automatically becomes a feature.  But you are correct, in this specific case at this specific time it does not offer much value other than a caution for the future

Reply:
`bundled-sqlcipher-vendored-**openssl**` which doesn't / won't depend on openssl...